### DISCUSSION TITLE
Recomended way to remove ManyToOne owning side relationship

### DISCUSSION THREAD

**Discussion by kennethsato** [mikro-orm/mikro-orm#5008] [Q&A]
I think this is a very basic question, but I couldn't find an answer for it.
I hope it's no repeated.

I've got an entity EnviaExibidor that owns an User

```
@Entity({ tableName: 'enviaexibidor' })
export default class EnviaExibidor {
(...)
  @ManyToOne(() => User)
  usuarioconferencia: User;
```

If i want to remove that user, what's the correct way to do it?
Should I just do the following?
`enviaexibidor.usuarioconferencia = null`

If so, then I must change my usuarioconferencia property to "User | null" on my EnviaExibidor entity? Like:

```
  @ManyToOne(() => User)
  usuarioconferencia: User | null;
```

Or is there a better way to do it?

**Comment by B4nan** [comment]
`enviaexibidor.usuarioconferencia = undefined` or `delete enviaexibidor.usuarioconferencia` will work too, of course you'd have to make that property optional (`usuarioconferencia?: User;`).

Or do you want to delete that target entity?

**Reply by kennethsato** [reply]
No! That's it!
I didn't know the delete option and it looks better to me. Chose that one!

Thanks again @B4nan
